Packages/Env/Sources/Env/UserPreferences.swift

```diff
@@ -154,4 +174,17 @@ public class UserPreferences: ObservableObject {
     copy.insert(isoCode, at: 0)
     recentlyUsedLanguages = Array(copy.prefix(3))
   }
+  
+  public static func getIntOfVisibility(_ vis: Models.Visibility) -> Int {
+    switch vis {
```

Comment:
I guess you could try to make Visibility enum an Int instead of String. Not sure we really use its String value for anything at the moment.

Reply:
The string value seems to be used for decoding from the JSON we get. I’m implementing a manual decoding, I’ll push it shortly, you can decide what you like better